java-checks/src/main/java/org/sonar/java/checks/helpers/QuickFixHelper.java

```diff
@@ -102,6 +113,72 @@ public static SyntaxToken previousToken(Tree tree) {
     return previousToken(parent);
   }
 
+  /**
+   * Returns the preceding variable in a mutli-variable declaration.
+   *
+   * @param current The target variable
+   * @return An Optional with the preceding variable, Optional.empty otherwise.
+   */
+  public static Optional<VariableTree> previousVariable(VariableTree current) {
+    Tree parent = current.parent();
```

Comment:
It would be simpler to maintain if the logic to find children was in a common method used by previousVariable and nextVariable. The method could return a list of trees without the `firstRelevantIndex`, it'ŝ not so useful.